src/civis/resources/_resources.py

```diff
@@ -648,11 +654,12 @@ def parse_api_spec(api_spec, api_version):
         if methods and classes.get(base_path) is None:
             cls = type(class_name, (Endpoint,), {})
             cls.__doc__ = (
-                "Examples\n"
-                "--------\n"
-                ">>> import civis\n"
-                ">>> client = civis.APIClient()\n"
-                f">>> client.{base_path}.{methods[0][0]}(...)"
+                f"Civis API ``/{base_path}`` endpoint:\n\n"
+                ".. code-block:: python\n\n"
+                "    import civis\n"
+                "    client = civis.APIClient()\n"
+                f"    # Call client.{base_path}.<method>(<arguments>) to make a request, e.g.:\n"  # noqa: E501
+                f"    client.{base_path}.{methods[0][0]}(...)\n\n"
```

Comment:
Making the example code snippet show up above rather than below the "Methods" table (as shown in the screenshots above), since some endpoints like "scripts" have a very long methods table which essentially buries this code snippet.